BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -404,7 +404,7 @@ send a "key_share" when initiating session resumption.
 In order to gain forward secrecy, this document recommends that server implementations SHOULD
 respond with a "key_share", to complete an ECDHE exchange on each session resumption.
 As a more performant alternative, server implementations MAY refrain from responding with a 
-"key_share" until a certain amount of time (e.g., measured in days) has passed since the last 
+"key_share" until a certain amount of time (e.g., measured in hours) has passed since the last 
 ECDHE exchange; this implies that the "key_share" operation would not occur for the presumed
 majority of session resumption requests occurring within a few hours, while still ensuring 
```

Comment:
If we say 'within a few hours" here we shouldn't say "days" for the analogous delay a couple lines previously.